Buildings/Controls/OBC/Utilities/PIDWithAutotuning/FirstOrderAMIGO.mo

```diff
@@ -350,13 +395,13 @@ at which point this block turns back to a PID controller but with tuned PID para
 <b>Note:</b> If an autotuning is ongoing, i.e., <code>inTunPro.y = true</code>,
 a request for performing autotuning will be ignored.
 </p>
-<h4>Guidance for settings the parameters</h4>
+<h4>Guidance for setting the parameters</h4>
 <p>
 The performance of the autotuning is affected by the parameters, including the
 typical range of control error, <code>r</code>, 
 the reference output for the tuning process, <code>yRef</code>, the lower value for
 the relay output, <code>yLow</code>, and the deadband, <code>deaBan</code>.
-The following procedure can be used for determining the values of those parameters. 
+The following procedure can be used to determine the values of those parameters. 
```

Comment:
For the first step, `Perform the simulation without enabling attuning and with ...`, I think it's best the clarify that it is a test run. The run should be stopped after the system is stable. We can then find the maximum and the minimum values of the measurement.

Reply:
I updated the first step accordingly.